### ISSUE TITLE
Error includes of null

### ISSUE THREAD

**Issue by smaznet** [chame1eon/jnitrace#9]
i got this error
```js 
ERROR: {'type': 'error', 'description': "TypeError: Cannot read property 'includes' of null", 'stack': "TypeError: Cannot read property 'includes' of null
   at jnitrace/src/main.ts:39:22
    at Array.forEach (<anonymous>)
    at Object.onLoaded (jnitrace/src/main.ts:38:26)
    at Object.doCallback (node_modules/jnitrace-engine/dist/index.js:127:13)
    at checkLibrary (node_modules/jnitrace-engine/dist/engine.js:44:9)
    at InvocationContext.<anonymous> (node_modules/jnitrace-engine/dist/engine.js:102:17)", 'fileName': 'jnitrace/src/main.ts', 'lineNumber': 39, 'columnNumber': 22}
```

**Comment by chame1eon** [comment]
Hi, Thanks for your bug report. I have had a look at the issue and believe it is to do with jnitrace/frida not being able to resolve the path of the native library that is being loaded. I put some error handling around that and have pushed the code to a new version 3.0.1. Can you upgrade to that package through pip and let me know if it fixes the problem? If not can you provide me with the APK or application name you are experiencing the issue with so I can try and reproduce it. Thanks

**Comment by smaznet** [comment]
that error gone but now app crashs

i cant share app on github
do you have telegram?

**Comment by chame1eon** [comment]
No sorry, is there anyway you can upload it to a file storage site and just send a link?

**Comment by smaznet** [comment]
No i don't want share my private app on GitHub
i should message in private

**Comment by chame1eon** [comment]
How about email to chame1eon@protonmail.com?

**Comment by chame1eon** [comment]
Thanks, got it. I'll have a look and get back to you

**Comment by chame1eon** [comment]
What libraries were you tracing to get the error?

**Comment by chame1eon** [comment]
Hi, so I have found a couple of bugs that I have fixed and pushed as version 3.0.2. However, I do not think that those bugs where causing the issue you had. Either way it seems to be working fine for me on an Android AVD emulator running Android 6, so it might be worth you trying an Android 6.0 emulator as well to see if that works. What device are you testing with?
